store/store.go

```diff
@@ -112,20 +112,33 @@ func (s *store) Index() uint64 {
 // If recursive is true, it will return all the content under the node path.
 // If sorted is true, it will sort the content by keys.
 func (s *store) Get(nodePath string, recursive, sorted bool) (*Event, error) {
-	s.worldLock.RLock()
-	defer s.worldLock.RUnlock()
+	var err *etcdErr.Error
```

Comment:
is there a reason this needs to be etcdErr.Error type? How about just error? Then we do not need to change the signature of `internalCreate` and it seems to be cleaner.